### PR TITLE
login

### PR DESCRIPTION
**Pull request by rushalipatelbt**
desc

### CODE REVIEW

File: CustomPullRequest/login.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//  CustomPullRequest
+//
+//  Created by Rushali's iMac on 03/10/24.
+//
+
+import SwiftUI
+
+struct login: View {
+    var body: some View {
+        Text("login")
```
Comment (harshiliosbt) [inline CustomPullRequest/login.swift:12]: change text login to hello

File: CustomPullRequest/login.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    }
+}
+
+#Preview {
+    login()
```
Comment (harshiliosbt) [inline CustomPullRequest/login.swift:17]: change

**Review by harshiliosbt — COMMENTED** [review]
check

**Review by rushalipatelbt — COMMENTED** [review]
test

**Review by harshiliosbt — APPROVED** [review]
done

**Pull request merged**